pkgs/development/python-modules/xlwt/default.nix

```
@@ -1,49 +1,41 @@
 {
   lib,
   buildPythonPackage,
-  fetchPypi,
-  nose,
+  fetchFromGitHub,
   setuptools,
-  pythonOlder,
+  pytestCheckHook,
+  six,
 }:
 
-buildPythonPackage rec {
+buildPythonPackage {
   pname = "xlwt";
-  version = "1.3.0";
+  version = "unstable-2018-09-16";
```

```suggestion
  version = "1.3.0-unstable-2018-09-16";
```

(Not sure if this is documented in the manual yet but we agreed to include the previous release version in `unstable` versions, or `0-unstable-…` if there is none.)

---

Wasn't aware of that, will update the version.